### ISSUE TITLE
What's is random seeds in training steps?

### ISSUE THREAD

**Issue by WillJohn** [XenderLiu/MalConv-Pytorch#3]
python3 train.py <config_file_path> <random_seed>

Is there any constrains or rule to set the random seed?
